inferno-framework/tutorial-test-kit
# FI-3166: Step 2 solution upgrade

## Summary
This PR updates the Inferno template version in step-2-solution branch. It includes the solution to step-2 of the tutorial.

## Testing
Run locally with Docker:
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - access_token: `SAMPLE_TOKEN`
    - Patient ID: `85`
- You should get the following results:

<img width="304" alt="image" src="https://github.com/user-attachments/assets/4875ed37-933a-4eeb-a7a1-e6cb56327b61">

Also run with Ruby.
